src/scripts.js

```diff
@@ -465,17 +498,8 @@ $('.toggle label').on('click', function() {
 
 $('.icons li img').on('click', function() {
   const $widget = $(this).data('type');
-  const $block = $('#user-admin-info');
-  if ($block.data('type') === $widget) {
-    $block.data('type', '').hide();
-  } else {
-    $block.show().children(`.${$widget}-inputs`).show().siblings().hide();
-    $block.data('type', $widget);
-  }
-});
-
-$('#user-admin-info').on('mouseleave', function() {
```

Comment:
I had this branch pulled down on my local computer and noticed that this functionality was gone. I am assuming you are thinking about reimplementing something of this kind? Right now when you click on User Info and then click to add hydration/sleep/activity data the forms overlap. I think the mouseleave was a good feature but maybe not the most accessible. I think it would be good to have the display toggle back to hidden when a person clicked out as well. And maybe we could combine them into enter key clicks or tabs as well as mouse click events 

Let me know what you are thinking for this